docs/faq.md

```diff
@@ -0,0 +1,43 @@
+# Frequently Asked Questions (FAQ)
+
+## 1. What is Feast?
+Feast (Feature Store) is an [open-source](https://github.com/feast-dev/feast) platform that helps teams define, manage, validate, and serve machine learning features to production environments, whether in batch or real-time.
+
+## 2. Who is Feast for?
+- **Data Scientists**: Easily define, store, and retrieve features for model development and deployment.
+- **MLOps Engineers**: Connect existing data infrastructure, ensuring data scientists can ship features to production without getting bogged down in engineering details.
+- **Data Engineers**: Maintain a single source of truth for feature data and definitions through a unified catalog.
+- **AI Engineers**: Scale AI applications by seamlessly integrating richer data, enabling efficient fine-tuning and serving of models.
+
+## 3. Is Feast an ETL/ELT system?
+No. Feast is not a general-purpose data pipelining system. Users typically rely on tools like [dbt](https://www.getdbt.com/) or other ETL/ELT solutions to handle upstream data transformations. Feast does support some [transformations](getting-started/architecture/feature-transformation.md), but it’s not meant to replace an end-to-end ETL/ELT workflow.
+
+## 4. Is Feast a data warehouse or a database?
+No. Feast does not replace your data warehouse. It orchestrates and serves data that typically resides in systems like BigQuery, Snowflake, DynamoDB, and Redis. It’s a lightweight layer to make features consistently available at training and serving time.
+
+## 5. Which problems does Feast not fully solve?
```

Comment:
Prior to this question a good question would be "What problems does Feast solve"?

This would quickly catch users attention.